executor/simple.go

```diff
@@ -1059,21 +1066,92 @@ func (e *SimpleExec) executeSetPwd(s *ast.SetPwdStmt) error {
 	return err
 }
 
-func (e *SimpleExec) executeKillStmt(s *ast.KillStmt) error {
-	conf := config.GetGlobalConfig()
-	if s.TiDBExtension || conf.CompatibleKillQuery {
-		sm := e.ctx.GetSessionManager()
-		if sm == nil {
-			return nil
-		}
+func (e *SimpleExec) executeKillStmt(ctx context.Context, s *ast.KillStmt) error {
+	sm := e.ctx.GetSessionManager()
+	if sm == nil {
+		return nil
+	}
+
+	if e.InCoprocessor {
+		logutil.BgLogger().Info("Kill in coprocessor", zap.Uint64("connID", s.ConnectionID), zap.Bool("query", s.Query))
```

Comment:
IP address added in latest [commit](https://github.com/pingcap/tidb/pull/17649/commits/fd317762de9ece8c544200a2b394372a8f817439). PTAL~

Server ID need to be sent with `coprocessor.Request`. It would affect performance if sending server ID on every request, and it would increase codes complexity if sending only in KILL commands.
It seems that it's expensive to logging server ID.

How do you think ? @SunRunAway

Reply:
LGTM